src/modules/sensors/vehicle_air_data/params.c

```diff
@@ -53,3 +53,12 @@ PARAM_DEFINE_FLOAT(SENS_BARO_QNH, 1013.25f);
  * @unit Hz
  */
 PARAM_DEFINE_FLOAT(SENS_BARO_RATE, 20.0f);
+
+/**
+ * air, fresh or salt water
+ *
+ * @min 0
+ * @max 2
```

Comment:
You can add the meaning of 0,1,2 in an enum like this: https://github.com/PX4/PX4-Autopilot/blob/cfc579542e1bad253c26fdc90fb432baf5ffcd07/src/drivers/gps/params.c#L43-L47

Reply:
3670f4722d16f331a9b23a5c6910b4a5c062fdea